app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com = command.split()
+    if com[1] != com[2]:
```

Comment:
com[1] and com[2] aren't declarative
use names like
`_, copied_file, file_to_copy = command.split()`